herbert_core/classes/MPIFramework/@JobExecutor/JobExecutor.m

```diff
@@ -343,6 +360,25 @@ function log_progress(this,step,n_steps,time_per_step,add_info)
             [ok,err] = obj.mess_framework_.send_message(message);
         end
 
+        function [ok,err,message]=receive_message(obj,from_task_id,varargin)
+            % Wrapper to receive_message to ease writing
+            [ok,err,message] = obj.mess_framework_.receive_message(from_task_id,varargin{:});
+        end
+
+        function obj = setup(obj)
+            % Function called once before entering do_job loop
```

Comment:
for these 2 functions, unclear if they will be filled in later, or redefined in a subclass or what.

Reply:
They are to be overridden by children, however, if I make them abstract, existing JEs will need to be modified with empty calls. This is a bit of a hack. Will clarify with comments.